integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -798,13 +798,25 @@ def _test_div_by_zero(ansi_mode, expr):
     ansi_conf = {'spark.sql.ansi.enabled': ansi_mode == 'ansi'}
     data_gen = lambda spark: two_col_df(spark, IntegerGen(), IntegerGen(min_val=0, max_val=0), length=1)
     div_by_zero_func = lambda spark: data_gen(spark).selectExpr(expr)
+    err_exp = 'java.lang.ArithmeticException' if is_before_spark_320() else \
+        'SparkArithmeticException'
+    # 3.3.0
+    #      org.apache.spark.SparkArithmeticException: divide by zero. XXXX
+    # 3.3.1
+    #      org.apache.spark.SparkArithmeticException: Division by zero. XXXX
+    # 3.4.0
+    #      org.apache.spark.SparkArithmeticException: [DIVIDE_BY_ZERO] Division by zero. XXXX
+    if is_before_spark_331():
+        err_mess = ': divide by zero'
+    elif not is_before_spark_331() and is_before_spark_340():
```

Comment:
`not is_before_spark_331()` is not needed because it's also in `if` branch.
Suggests:
```
if is_before_spark_320():
  error_message = 
elif is_before_spark_331():
  error_message = 
elif is_before_spark_340():
  error_message = 
elif:
  error_message = 

assert_gpu_and_cpu_error(... , error_message = )
```
And the comments can be removed.

Reply:
done